cleanlab/datalab/internal/issue_manager/regression/label.py

```diff
@@ -43,16 +34,98 @@ class RegressionLabelIssueManager(IssueManager):
     datalab :
         A Datalab instance.
 
-    ... :
-        ...
-
     clean_learning_kwargs :
         Keyword arguments to pass to the :py:meth:`CleanLearning <cleanlab.classification.CleanLearning>` constructor.
 
-    health_summary_parameters :
-        Keyword arguments to pass to the :py:meth:`health_summary <cleanlab.dataset.health_summary>` function.
+    """
+
+    description: ClassVar[
+        str
+    ] = """Examples whose given label is estimated to be potentially incorrect
+    (e.g. due to annotation error) are flagged as having label issues.
     """
 
     issue_name: ClassVar[str] = "label"
+    verbosity_levels = {
+        0: [],
+        1: [],
+        2: [],
+        3: [],  # TODO
+    }
+
+    def __init__(
+        self,
+        datalab: Datalab,
+        clean_learning_kwargs: Optional[Dict[str, Any]] = None,
+        health_summary_parameters: Optional[Dict[str, Any]] = None,
+    ):
+        super().__init__(datalab)
+        self.cl = CleanLearning(**(clean_learning_kwargs or {}))
+
+    @staticmethod
+    def _process_find_label_issues_kwargs(kwargs: Dict[str, Any]) -> Dict[str, Any]:
+        """Searches for keyword arguments that are meant for the
+        CleanLearning.find_label_issues method call
+
+        Examples
+        --------
+        >>> from cleanlab.datalab.internal.issue_manager.regression.label import LabelIssueManager
+        >>> RegressionLabelIssueManager._process_find_label_issues_kwargs({'coarse_search_range': [0.1, 0.9]})
+        {'coarse_search_range': [0.1, 0.9]}
+        """
+        accepted_kwargs = [
+            "uncertainty",
+            "coarse_search_range",
+            "fine_search_size",
+            "save_space",
+            "model_kwargs",
+        ]
+        return {k: v for k, v in kwargs.items() if k in accepted_kwargs and v is not None}
+
+    def find_issues(
+        self,
+        **kwargs,
+    ) -> None:
+        """Find label issues in the datalab."""
+
+        RegressionLabelIssueManager.issue_score_key = "label_score"
```

Comment:
Doesn't the [metaclass `IssueManagerMeta`](https://github.com/cleanlab/cleanlab/blob/14d1a86d4375beb17168eafdea88c84fd73c9a65/cleanlab/datalab/internal/issue_manager/issue_manager.py#L78) already handle this properly? Can you add a test case to confirm this?

It should work without this line added explicitly.

Reply:
Great catch! Fixed in https://github.com/cleanlab/cleanlab/pull/796/commits/49635a785fd0e301fe7b6869f30f708b6b2514ad